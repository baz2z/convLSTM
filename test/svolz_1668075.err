Traceback (most recent call last):
  File "/mnt/qb/work/butz/svolz67/dev/convLSTM/test/./testAll-speed.py", line 281, in <module>
    loss170 = calcLoss(model, 100, 20, 170, dataLoader)
  File "/mnt/qb/work/butz/svolz67/dev/convLSTM/test/./testAll-speed.py", line 231, in calcLoss
    model.load_state_dict(torch.load("model.pt", map_location=device))
  File "/mnt/qb/work/butz/svolz67/envs/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1497, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for Forecaster:
	size mismatch for init.weight: copying a param with shape torch.Size([4, 1, 1, 1]) from checkpoint, the shape in current model is torch.Size([10, 1, 1, 1]).
	size mismatch for init.bias: copying a param with shape torch.Size([4]) from checkpoint, the shape in current model is torch.Size([10]).
	size mismatch for encoder_layers.block_0.conv.weight: copying a param with shape torch.Size([16, 8, 3, 3]) from checkpoint, the shape in current model is torch.Size([40, 20, 3, 3]).
	size mismatch for encoder_layers.block_0.conv.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([40]).
	size mismatch for encoder_layers.block_1.conv.weight: copying a param with shape torch.Size([16, 8, 3, 3]) from checkpoint, the shape in current model is torch.Size([40, 20, 3, 3]).
	size mismatch for encoder_layers.block_1.conv.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([40]).
	size mismatch for decoder_layers.block_0.conv.weight: copying a param with shape torch.Size([16, 4, 3, 3]) from checkpoint, the shape in current model is torch.Size([40, 10, 3, 3]).
	size mismatch for decoder_layers.block_0.conv.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([40]).
	size mismatch for decoder_layers.block_1.conv.weight: copying a param with shape torch.Size([16, 8, 3, 3]) from checkpoint, the shape in current model is torch.Size([40, 20, 3, 3]).
	size mismatch for decoder_layers.block_1.conv.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([40]).
	size mismatch for read.weight: copying a param with shape torch.Size([1, 4, 1, 1]) from checkpoint, the shape in current model is torch.Size([1, 10, 1, 1]).
