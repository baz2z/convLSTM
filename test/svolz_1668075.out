/mnt/qb/work/butz/svolz67/dev/convLSTM/test
JOB DATA
JobId=1668075 ArrayJobId=1668075 ArrayTaskId=1 JobName=review.sh
   UserId=svolz67(4458) GroupId=butz(4032) MCS_label=N/A
   Priority=13634 Nice=0 Account=butz QOS=normal
   JobState=RUNNING Reason=None Dependency=(null)
   Requeue=1 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
   RunTime=00:00:01 TimeLimit=3-00:00:00 TimeMin=N/A
   SubmitTime=2022-08-01T09:16:35 EligibleTime=2022-08-01T09:16:35
   AccrueTime=Unknown
   StartTime=2022-08-01T09:16:35 EndTime=2022-08-04T09:16:35 Deadline=N/A
   PreemptEligibleTime=2022-08-01T09:17:35 PreemptTime=None
   SuspendTime=None SecsPreSuspend=0 LastSchedEval=2022-08-01T09:16:35
   Partition=gpu-2080ti AllocNode:Sid=bg-slurmb-login1:13061
   ReqNodeList=(null) ExcNodeList=(null)
   NodeList=slurm-bm-27
   BatchHost=slurm-bm-27
   NumNodes=1 NumCPUs=8 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   TRES=cpu=8,mem=3G,node=1,billing=2,gres/gpu=1
   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
   MinCPUsNode=1 MinMemoryNode=3G MinTmpDiskNode=0
   Features=(null) DelayBoot=00:00:00
   OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
   Command=/mnt/qb/work/butz/svolz67/dev/convLSTM/test/review.sh
   WorkDir=/mnt/qb/work/butz/svolz67/dev/convLSTM/test
   StdErr=/mnt/qb/work/butz/svolz67/dev/convLSTM/test/svolz_1668075.err
   StdIn=/dev/null
   StdOut=/mnt/qb/work/butz/svolz67/dev/convLSTM/test/svolz_1668075.out
   Power=
   CpusPerTres=gpu:8
   TresPerNode=gpu:1
   MailUser=sebastian.volz@student.uni-tuebingen.de MailType=FAIL
   NtasksPerTRES:0

node:
slurm-bm-27.novalocal
RUN Script
